config/kubernetes-sigs/sig-contributor-experience/teams.yaml

```
@@ -22,6 +22,8 @@ teams:
     members:
     - coderanger
     - jberkus
+    - sreeram-venkitesh
+    - fykaa
```

```suggestion
    - coderanger
    - fykaa
    - jberkus
    - sreeram-venkitesh
```
the list needs to be alpha-sorted (I'm hoping indentation is okay via GitHub suggestions)